src/seedu/addressbook/commands/AddCommand.java

```diff
@@ -2,6 +2,7 @@
 
 import seedu.addressbook.data.exception.IllegalValueException;
 import seedu.addressbook.data.person.*;
+import seedu.addressbook.data.person.address.*;
```

Comment:
I understand, even if everything imported by * are only the necessary classes, it is still better to list out all the classes right? So in this case, I should also change the previous line `import seedu.addressbook.data.person.*;`